scripts/llm/ptq.py

```diff
@@ -30,7 +33,11 @@ def get_args():
     parser.add_argument("-cpp", "--calibration_pp", "--calib_pp", type=int, default=1)
     parser.add_argument("-itp", "--inference_tp", "--tensor_parallelism_size", type=int, default=1)
     parser.add_argument("-ipp", "--inference_pp", "--pipeline_parallelism_size", type=int, default=1)
+    parser.add_argument("-nodes", "--num_nodes", type=int, default=1)
     parser.add_argument('-out', '--export_path', '--output_path', type=str, help='Path for the exported engine')
+    parser.add_argument(
+        "--save_as_torch", action="store_true", help="Save `-out` as a Torch checkpoint instead of TRT-LLM"
```

Comment:
Thoughts on changing to `--export_format` which is default as `trtllm` but can also be `nemo` or in future `hf`